test/Conversion/FIRRTLToHW/lower-to-hw.mlir

```diff
@@ -670,13 +667,19 @@ firrtl.circuit "Simple"   attributes {annotations = [{class =
     // CHECK-NEXT:    sv.ifdef "RANDOMIZE_REG_INIT" {
     // CHECK-NEXT:      %[[RANDOM:.+]] = sv.reg sym @[[RANDOM_SYM:[_A-Za-z0-9]+]] {{.+}}
     // CHECK-NEXT:    }
+    // CHECK-NEXT:    sv.ifdef "FIRRTL_BEFORE_INITIAL" {
```

Comment:
Should this be in the initial block?  "before" doesn't make sense for anything which expands to  code in this context.

Reply:
I agree but SFC assumes `FIRRTL_BEFORE_INITIAL` is expanded in non-procedural region. 
https://github.com/chipsalliance/firrtl/blob/2eddce1a40709381b4ae9cc952708e7d6aa7b969/src/test/scala/firrtlTests/VerilogEmitterTests.scala#L1381-L1382